src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -2113,8 +2121,13 @@ void Interpolate::initSupportedPrimitiveDescriptors() {
         pushDesc(LayoutType::nspc, undef, true, true);
         pushDesc(LayoutType::ncsp, undef, true, true);
         canUseAclExecutor = !supportedPrimitiveDescriptors.empty();
-        if (canUseAclExecutor)
+        if (canUseAclExecutor) {
             return;
+        }
+        //if ACL executor is not selected then f16 should not be used
+        if (inputPrecision == ov::element::f16) {
```

Comment:
Ref Interpolate supports fp32 only. So we need to fallback on fp32 in any case, not just for fp16

Reply:
Fixed